.github/workflows/push-docker-image.yml

```diff
@@ -25,10 +25,14 @@ jobs:
         file: ./facilitator/Dockerfile
```

Comment:
This `build` step should be removed since you added the one that invokes `build.sh`